packages/core/src/tests/panes/panes.e2e.ts

```diff
@@ -142,6 +142,13 @@ test.describe('pane', () => {
     await expect(page).toHaveScreenshot();
   });
 
+  regressionTest('layout, inline, collapsed, relative', async ({ page }) => {
```

Comment:
You need to regenerate the snapshots
```suggestion
  regressionTest('layout, inline, collapsed, responsive', async ({ page }) => {
```